sycl/include/sycl/info/info_desc.hpp

```diff
@@ -17,6 +17,7 @@
 #include <sycl/aspects.hpp>
 #include <sycl/detail/type_traits.hpp>
 #include <sycl/ext/oneapi/experimental/device_architecture.hpp>
+#include <sycl/ext/oneapi/matrix/static-query-use.hpp>
```

Comment:
I am wondering whether we should bring `combination` and `matrix_type` definition here so we get independent from this include to the static query.

Reply:
yes, how about https://github.com/intel/llvm/blob/sycl/sycl/include/sycl/ext/oneapi/matrix/utils.hpp?

Reply:
I did, but it is unclear for developer at first sight to understand that utils.hpp is used in static-query-use.hpp. Also "utils" means some additional utilities for implementation details, but here we add two public-visible data types.
Does it make sense to create something like query-types.hpp or something like that and include directly to static-query-use.hpp?

Reply:
Please do that

Reply:
> does it help if we rename utils to matrix-utils? On a second thought, including static-query-use.hpp in there is fine as this shows the two can be used together.

Don't think about matrix-utils.hpp as we have information about matrix in the full path: sycl/ext/oneapi/matrix/utils.hpp
I just don't think "utils" is a suitable name for a header with two public types.

If you want, we can leave everything as it is now, overwise I suggest something like https://github.com/intel/llvm/pull/11250#discussion_r1333330037

Reply:
@dkhaldi with `combination` and `matrix_type` in utils.hpp, there are redefinition error. Moved back to using the original usage of static-query-use.hpp, but some LIT tests started to fail with weird error. Can you please advise how to fix this?

Reply:
Looks like resolved.